### ISSUE TITLE
Supports infinite canvas

### ISSUE THREAD

**Issue by fgiering** [dineug/erd-editor#369]
**Is your feature request related to a problem? Please describe.**
If you arrange your node-setup and you want to put nodes in the upper left corner, but the canvas ends their, than there is no possibility to get space for the node except from moving all nodes individually. In VS Code you can set the node "out of bounds", but this will lack on png export

**Describe the solution you'd like**
Move all selected nodes together or move the canvas withou the nodes.

![grafik](https://github.com/dineug/erd-editor/assets/35344457/4b25f863-0e6b-42f9-8998-f5842ce26040)

**Comment by dineug** [comment]
Hold Ctrl to select multiple tables and move them all at once.

**Comment by fgiering** [comment]
I do not think this is a solution, if you have a few houndred nodes this will be an enormous time consuming process. Needs a select all or move canvas/artboard.

**Comment by dineug** [comment]
Am I missing something here? Could you explain it in a bit more detail?

https://docs.erd-editor.io/docs/guide/guides/table-related-functions#multiple-selection

<img width="702" height="712" alt="Image" src="https://github.com/user-attachments/assets/df1bc07c-c633-4948-8073-867fa16c9c32" />

<img width="1046" height="593" alt="Image" src="https://github.com/user-attachments/assets/ac9704b0-0eb9-4dab-b73c-04dc61bd4336" />

**Comment by fgiering** [comment]
Ah. Okay. This is shortcut very uncommon (need to Press "Ctrl + Alt + A on Windows). Why not intercepting Ctrl + A?

An then you need to hold "Ctrl" to move all along. Why not moving al selected nodes if one of the selected is beeing grabbed? (Would be more common and relfect the behaviour in other node tools.)

The Artboard is still not growing if I move nodes out of bounds. Isnt this needed?

(Sidefact: "Alt" during moving a bunch of nodes is copying/duplicating them but is missing relationships.)

**Comment by dineug** [comment]
Thank you for the detailed explanation.

I'll give it some thought, as the infinite canvas also needs to take the minimap into consideration.

**Comment by fgiering** [comment]
Awesome solution. Thanks!
